search/indexing_api.py

```diff
@@ -249,7 +274,7 @@ def sync_comments(serialized):
 
     for item in serialized:
         sync_comment_model(
-            channel_name=item["channel_title"],
+            channel_name=item["channel_name"],
```

Comment:
Good catch!

Reply:
I think some erroneously created channels will need to be manually deleted from admin.

Reply:
Ah yes, I bet that's where some of the extra `Channel` objects are coming from